### PR TITLE
Silent prompt changes

### PR DESCRIPTION
**Pull request by rgomezp**
### Motivation
Chrome will be changing the push notification opt-in process in some cases. These changes are being built and tested now and will be broadly rolled out with Chrome 80, with an estimated release date of Feb 4, 2020.

### Implementation
Change the label for native prompt “Attempt to show a native browser permission prompt immediately upon user visit (not recommended)

Change the label for slide prompt “Automatically show the Slide Prompt before prompting users to subscribe”

Move native prompt option down to bottom

Show warning banner if native turned on “We strongly recommend not immediately prompting users with a native prompt as most browsers will hide this prompt if users frequently hit deny. More information.” (link to docs)

---
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-wordpress-plugin/224)

### CODE REVIEW

File: onesignal-public.php [hunk 1]
```diff
@@ -75,20 +75,24 @@ public static function onesignal_header()
             echo 'OneSignal.setDefaultNotificationUrl("'.esc_url($onesignal_wp_settings['default_url']).'");';
         } else {
             echo 'OneSignal.setDefaultNotificationUrl("'.esc_url(get_site_url())."\");\n";
-        } ?>
+        } 
+        
+        if ($onesignal_wp_settings['prompt_auto_register'] === true) {
```
Comment (jkasten2) [inline onesignal-public.php]: These changes call methods directly and do not check for `use_custom_sdk_init` like the `oneSignal_options` was using before. I recommend putting them in the `if (!use_custom_sdk_init)` block, just after OneSignal init

Comment (rgomezp) [reply onesignal-public.php]: RESOLVED

File: onesignal-public.php [hunk 2]
```diff
@@ -306,6 +298,15 @@ public static function onesignal_header()
               <?php
                 }
             }
+        
+            if ($onesignal_wp_settings['prompt_auto_register'] === true) {
+                    echo "OneSignal.showSlidedownPrompt();";
```
Comment (jkasten2) [inline onesignal-public.php]: Just noticed we also have a call to `OneSignal.showHttpPrompt();` above if `use_slidedown_permission_message_for_https` is enabled. However I think we can clean this up now as I don't see us reading the key but please double check to be sure.

Comment (rgomezp) [reply onesignal-public.php]: Cleaned this up

**Pull request merged**